src/style.css

```diff
@@ -1,3 +1,63 @@
+* {
+  margin: 0;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.navigation {
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  padding-left: 50px;
+  align-items: center;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  margin-right: 31px;
+}
+
+.nav-logo {
+  width: 40px;
+  margin-top: 10px;
+  height: 40px;
+}
+
+.nav-list {
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  align-items: stretch;
+  text-transform: uppercase;
+  margin: 10px auto;
+}
+
+.is-active {
+  color: black;
+  text-decoration: none;
+  display: block;
+  padding: 10px;
+  flex-shrink: 0;
+}
+
```

Comment:
You should also add the blue line under the first link - apple.
After applying changes related to class naming I mentioned before try to do something linke this:
create ".nav__link--active" selector in which you are going to set the color to blue and position relative (we need this position in order to place the blue line correctly)

Below this selector block create an after pseudo element like this: `.nav__link--active::after` - this is going to be the blue line. Inside set all necessary properties like content: ""; width: 100% etc. and remember about position: absolute; 

;)